astroquery/gemini/core.py

```diff
@@ -156,9 +156,10 @@ def query_object(self, objectname, radius=0.3*units.deg):
         return self.query_criteria(objectname=objectname, radius=radius)
 
     @class_or_instance
-    def query_criteria(self, coordinates=None, radius=0.3*units.deg, pi_name=None, program_id=None, utc_date=None,
+    def query_criteria(self, *args, coordinates=None, radius=0.3*units.deg, pi_name=None, program_id=None, utc_date=None,
```

Comment:
I think you may have misinterpreted my change.  For example:

```
def old_call(a, b, c="c"):
    print("Old Call")
    print("a: %s, b: %s, c: %s" % (a, b, c))

def new_call(a, b, *args, c="c", **kwargs):
    print("New Call")
    print("a: %s, b: %s, c: %s" % (a, b, c))


if __name__ == "__main__":
    old_call("a", "b", "c")
    new_call("a", "b", "c")
```

```
$ python argstest.py
Old Call
a: a, b: b, c: c
New Call
a: a, b: b, c: c
```

Reply:
I've renamed them with the rawquery name prefix and moved args to the back.  I ended up putting orderby sandwiched between them to prevent people using it via position.  I didn't want future search terms to result in orderby sitting in the middle of the positonal args list.

Prior, orderby was being picked up by the *args, so the explicit add and handling is really just making the signature match the docs.

Reply:
Thanks. We could also switch to keyword only args, definitely in the next refactoring release, but given the `gemini` module is relatively new, if you want you can make that change any time you wish.